fixtures/eurolite/led-theatre-cob-100-ww.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Theatre COB 100 WW",
+  "categories": ["Blinder"],
+  "meta": {
+    "authors": ["Benjamin Blaikner/ Theater der Mitte"],
+    "createDate": "2024-07-18",
+    "lastModifyDate": "2024-07-18"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.at/de/mpn41602010-eurolite-led-theatre-cob-100-ww.html"
+    ],
+    "productPage": [
+      "https://www.steinigke.at/de/mpn41602010-eurolite-led-theatre-cob-100-ww.html"
+    ]
+  },
+  "physical": {
+    "dimensions": [240, 320, 310],
+    "weight": 2.75,
+    "power": 85,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 3100
+    }
+  },
+  "availableChannels": {
+    "Warm White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
```

Comment:
Do we want to explicitly declare "Warm White" here?

Reply:
I guess it would be good!
As I am doing this for the first time: Should I do the corrections or are you doing them? And when will the file be ready to download? Thanks a lot!!